### PR TITLE
Add WebRTC clusters definitions

### PR DESCRIPTION
**Pull request by andyg-apple**
2 interlinked WebRTC clusters defined. The spec is in the cameras branch of the specs repo.
There is a StreamType enum added to one of the clusters (used in both) that is defined in the spec in a different cluster (AVStreamManagement). I have defined in here to unblock for now, and added an issue to remove as soon as AVStreamManagement cluster definition is merged.

### CODE REVIEW

File: src/app/zap-templates/zcl/zcl-with-test-extensions.json [hunk 1]
```diff
@@ -130,6 +130,8 @@
         "washer-controls-cluster.xml",
         "water-heater-management-cluster.xml",
         "water-heater-mode-cluster.xml",
+        "webrtc-requestor-cluster.xml",
+        "webrtc-provider-cluster.xml",
```
Comment (pidarped) [inline src/app/zap-templates/zcl/zcl-with-test-extensions.json:134]: The cluster attributes need to be listed in this file as well.

File: src/app/zap-templates/zcl/zcl.json [hunk 2]
```diff
@@ -124,6 +124,8 @@
         "washer-controls-cluster.xml",
         "water-heater-management-cluster.xml",
         "water-heater-mode-cluster.xml",
+        "webrtc-requestor-cluster.xml",
+        "webrtc-provider-cluster.xml",
```
Comment (pidarped) [inline src/app/zap-templates/zcl/zcl.json:128]: The cluster attributes need to be listed in this file too.

**Comment by gmarcosb** [comment]
I think we may be trampling each others' work 😅

I originally had this PR as one big one, as they are all somewhat interlinked: https://github.com/project-chip/connectedhomeip/pull/35773

But @bzbarsky-apple requested that I split it up to one-cluster-per-PR, and so I've done that & have a pending PR for one of the webrtc clustes: https://github.com/project-chip/connectedhomeip/pull/35773

(was going to get that in first before doing the 2nd)

**Comment by andyg-apple** [comment]
ok, killing this one and will create a new pr just with requestor

**Pull request closed without merging**